cg/apps/demultiplex/sbatch.py

```diff
@@ -3,9 +3,9 @@
 DEMULTIPLEX_COMMAND = {
     "bcl2fastq": """
 log "singularity exec --bind \
-/home/proj/{environment}/demultiplexed-runs,\
-/home/proj/{environment}/flow_cells,\
-/home/proj/{environment}/flow_cells/'$SLURM_JOB_ID':/run/user/$(id -u) \
+/home/proj/{environment}/sequencing_data/illumina_novaseq_flow_cells/demultiplexed-runs,\
```

Comment:
This would not be needed anymore- basically, the inputdir is the output dir. We likely need to refactor some flows first to accomodate this change.

Or if we like to have this separation we could always make a "demultiplexed_data" folder within the flow cells dir.

Reply:
Yes, this PR can be seen as an intermediate solution. Merging the directories would require some more re-writing which I suggest should be it's own issue.